src/ShipEngine.php

```diff
@@ -28,34 +35,42 @@
 class ShipEngine
 {
     /**
-     * Request Factory
-     *
-     * @var Api\RequestFactory
+     * @var Factory
      */
-    protected $requestFactory;
+    protected $addressFactory;
 
     /**
-     * @var Address\Factory
+     * @var Client
      */
-    protected $addressFactory;
+    protected $client;
 
     /**
      * ShipEngine constructor.
      *
      * @param string                     $apiKey
-     * @param Address\FormatterInterface $addressFormatter
+     * @param FormatterInterface $addressFormatter
+     * @param array                      $options
      */
-    public function __construct(string $apiKey, Address\FormatterInterface $addressFormatter)
+    public function __construct(string $apiKey, FormatterInterface $addressFormatter, array $options = [])
     {
-        $this->requestFactory = new Api\RequestFactory($apiKey);
-        $this->addressFactory = new Address\Factory($addressFormatter);
+        $defaultOptions = [
+            'base_uri' => "https://api.shipengine.com/v1/",
```

Comment:
This could be single quotes since no variable join

Reply:
Done!